[ONNX][dynamo_export] Turn off opmath type promotion for div

BowenBao opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #119112

Skip opmath promotion for `_prims_common.ELEMENTWISE_TYPE_PROMOTION_KIND.INT_TO_FLOAT` as well.
Fixes https://github.com/pytorch/pytorch/issues/118941

BowenBao committed:
[ONNX][dynamo_export] Turn off opmath type promotion for div

[ghstack-poisoned]

thiagocrepaldi reviewed (approved):
LGTM, but there are several `AssertionError: Tensor-likes are not close!` failures for the ONNX CI

BowenBao committed:
Relax tolerance for masked.var on "[ONNX][dynamo_export] Turn off opmath type promotion for div"


Skip opmath promotion for `_prims_common.ELEMENTWISE_TYPE_PROMOTION_KIND.INT_TO_FLOAT` as well.
Fixes https://github.com/pytorch/pytorch/issues/118941


[ghstack-poisoned]

BowenBao commented:
@pytorchbot merge
